src/WebSdk/ProjectSystem/content/Microsoft.NET.Sdk.Web.ProjectSystem.DefaultGlobalUsing.cs

```diff
@@ -0,0 +1,15 @@
+global using System;
```

Comment:
This list is based on https://github.com/dotnet/aspnetcore/issues/32451#issue-877008542. Though I've removed EF from the list.